docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -0,0 +1,352 @@
+
+# Verify that change has been applied
+$ aws route53 get-change --id ${CHANGEID?} | jq '.ChangeInfo.Status'
+"INSYNC"
+```
+</TabItem>
+</Tabs>
+
+The first request to Teleport's API will take a bit longer because it gets
+a cert from [Letsencrypt](https://letsencrypt.org).
+Teleport will respond back with a discovery info:
+
+```bash
+$ curl https://tele.example.com/webapi/ping
+
+{"server_version":"5.0.0-dev","min_client_version":"3.0.0"}
+```
+
+## Step 2/3 Create a local admin
+
+Local users are reliable fallback for cases when SSO provider is down.
+Let's create a local admin `alice` who has access to Kubernetes group `system:masters`.
+
+Save this role as `member.yaml`:
+
+```yaml
+kind: role
+version: v3
+metadata:
+  name: member
+spec:
+  allow:
+    # This field is used for SSH logins. You have to keep logins non-empty random value
+    # for Kubernetes to work until we fix it.
+    logins: ['keep-this-value-here']
+    kubernetes_groups: ["system:masters"]
+```
+
+Create the role and add a user:
+
+```bash
+# To create a local user, we are going to run Teleport's admin tool tctl from the pod.
+$ POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+
+# Create a role
+$ kubectl exec -i ${POD?} tctl -- create -f < member.yaml
+
+# Generate an invite link for the user.
+$ kubectl exec -ti ${POD?} tctl -- users add alice --roles=member
+
+User "alice" has been created but requires a password. Share this URL with the user to
+complete user setup, link is valid for 1h:
+
+https://tele.example.com:443/web/invite/random-token-id-goes-here
+
+NOTE: Make sure tele.example.com:443 points at a Teleport proxy which users can access.
+```
+
+Let's install `tsh` and `tctl` on Linux.
+For other install options, check out [install guide](../installation.mdx)
+
+<Tabs>
+<TabItem label="Open Source">
+```bash
+$ curl -L -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ sudo mv teleport/tsh /usr/local/bin/tsh
+$ sudo mv teleport/tctl /usr/local/bin/tctl
+```
+</TabItem>
+<TabItem label="Enterprise">
+```bash
+$ curl -L -O https://get.gravitational.com/teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+$ sudo mv teleport/tsh /usr/local/bin/tsh
+$ sudo mv teleport/tctl /usr/local/bin/tctl
+```
+</TabItem>
+</Tabs>
+
+Try `tsh login` with a local user. Use a custom `KUBECONFIG` to prevent override
+of the default one in case if there is a problem.
+
+```bash
+$ KUBECONFIG=${HOME?}/teleport.yaml tsh login --proxy=tele.example.com:443 --user=alice
+```
+
+Teleport updated `KUBECONFIG` with a short-lived 12 hour certificate.
+
+```bash
+$ tsh kube ls
+
+Kube Cluster Name Selected
+----------------- --------
+tele.example.com  *
+
+# Once working, remove the KUBECONFIG= override to switch to teleport
```

Comment:
Not sure what we're trying to explain to people with this line?

Reply:
It was easy for me to corrupt my kubeconfig when testing this, so wanted to give a tip to users

Reply:
Yeah, I just didn't quite understand how that would involve switching to Teleport when we'd already overridden the `KUBECONFIG`. I think it's probably clear enough.